.github/workflows/test.yaml

```diff
@@ -33,6 +33,21 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install requirements
       run: pip install -r requirements_for_test.txt
+    - name: Setup upterm session
+      uses: lhotari/action-upterm@v1
+      with:
+        ## limits ssh access and adds the ssh public key for the user which triggered the workflow
+        limit-access-to-actor: true
+        ## limits ssh access and adds the ssh public keys of the listed GitHub users
+        limit-access-to-users: jzbahrai
```

Comment:
What does this do?  Should your username be in here?

Reply:
ah i did it for debugging, it allows you to ssh into the box.